client/src/components/profileInfoEdit/ProfileClearModal.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Delete Alert Modal component display a modal and ask for confirmation when the delete button is clicked
+ */
+
+import React from 'react';
+import toast from 'react-hot-toast';
+import { Button, Modal } from 'react-bootstrap';
+import { useSelector, useDispatch } from 'react-redux';
+import { successMessageCreator } from '../../notification/notificationReduxFunctions';
+import { clearTeam, deleteGHPages } from '../../actions/team';
+
+const ProfileClearModal = ({ clearAlert, setClearAlert }) => {
+  const { teamId } = useSelector((state) => state.team);
+  const dispatch = useDispatch();
+
+  const HandleClear = () => {
+    const accessToken = localStorage.getItem('GH_access_token');
+    try {
+      dispatch(deleteGHPages(teamId, accessToken));
+    } catch (error) {
+      toast.error('GitHub Pages doesnt exist');
```

Comment:
The action should be using an error message toast to say the same thing, so you don't need a toast here?